baldrick/github/github_api.py

```diff
@@ -474,6 +474,64 @@ def is_closed(self):
 
 class PullRequestHandler(IssueHandler):
 
+    # https://developer.github.com/v3/checks/runs/#create-a-check-run
+    def set_check(self, name, summary, commit_hash='head', details_url=None,
+                  status='queued', conclusion='neutral'):
+        """
+        Set check status.
+
+        .. note:: This method does not provide API access to full
+                  check run capability (e.g., Markdown text, annotation,
+                  image). Add them as needed.
+
+        Parameters
+        ----------
+        name : str
+            Name of the check.
+
+        summary : str
+            Summary of the check run.
+
+        commit_hash: { 'head' | 'base' }
```

Comment:
Can you add ``, optional`` here and below

Reply:
Done